src/components/ui/Header.jsx

```diff
@@ -1,7 +1,11 @@
 function Header() {
   return (
     <header>
-      <nav>Header</nav>
+      <div className="flex justify-center">
```

Comment:
The designs show it aligned to the left, removing justify-center should achieve that.
```suggestion
      <div className="flex">
```